[deckhouse] fix module desc docs and omit empty module`s desc (#12249)
deckhouse/deckhouse: docs/documentation/pages/module-development/STRUCTURE.md

The file might not be present, but it is recommended to fill it in. Most of the metadata will be available in the [Module](../../cr.html#module) object in the cluster. The Module object will be created automatically after the module source (resource [ModuleSource](../../cr.html#modulesource)) is configured and synchronization is successful.

Parameters that can be used in `module.yaml`:

> - `namespace` - *String.* Namespace where the module parts work.
> - `subsystems` - *Array of strings.* List of subsystems the module belongs to.
> - `descriptions` — *Object.* Arbitrary text description of the module's purpose.
>   - `en` - *String.* English description.

Reviewer: ```suggestion
- `namespace` - *String.* The namespace where the module components will be deployed..
- `subsystems` - *Array of strings.* List of subsystems the module belongs to.
- `descriptions` — *Object.* Arbitrary text description of the module's purpose.
  - `en` - *String.* Description in English.
```
